README.md

```diff
@@ -4,57 +4,59 @@ ea_restaurant_data_loader is a microservice as part of [ea_restaurant](https://g
 
 The main objective of this project is the introduction of functional programming with elixir and its web framework phoenix. This project will impact the whole application since it works in asynchronous way it didn't depend of the flow of the whole app. Also This project can improve the whole application as the functional programming can offer several benefits such as improved readability and maintainability, enhance testability and debugging, and increase performance and scalability.
 
-## Project Setup
+## Ea_Restaurant_Data Loader Setup:
 
-In Order to build a Phoenix application, we will need a few dependencies installed in our Operating System:
+In order to setup and compile or run test in ea_restaurant_data_loader project you need to follow the following steps:
```

Comment:
It doesn't specify in what OS these steps will be performed.

Reply:
Done.